addons/linecompass/config.cpp

```diff
@@ -0,0 +1,17 @@
+#include "script_component.hpp"
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {"diwako_dui_main", "diwako_dui_radar"};
```

Comment:
I see no reference for the radar component anymore, you sure radar is needed here?